posthog/api/decide.py

```diff
@@ -150,7 +150,9 @@ def get_decide(request: HttpRequest):
             team = user.teams.get(id=project_id)
 
         if team:
-            feature_flags = get_overridden_feature_flags(team.pk, data["distinct_id"], data.get("groups", {}))
+            feature_flags = get_overridden_feature_flags(
```

Comment:
If we're touching /decide endpoint, that should be a separate PR to ensure easy control if something goes wrong. This single endpoint breaking can bring down all of posthog-js ingestion in nasty ways.